atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TokenRangeResolution.java

```diff
@@ -55,7 +55,7 @@ public static boolean viewsAreConsistent(Set<Set<TokenRange>> tokenRangeViews) {
     @Value.Immutable
     interface IdentityAgnosticRanges {
         @Value.Parameter
-        List<IdentityAgnosticRange> identityAgnosticRanges();
+        ImmutableSet<IdentityAgnosticRange> identityAgnosticRanges();
     }
```

Comment:
nit: is the explicit use of immutableSet on purpose?

Reply:
yes this is explicit to be specific that this is immutable and avoid an extra collection copy in `ImmutableIdentityAgnosticRanges` construction since we're now collecting the token range views to `ImmutableSet`s and passing those to  `ImmutableIdentityAgnosticRanges::of`
https://github.com/palantir/atlasdb/blob/8b4ff2da6f748826589c2e24bdb9014cd76fc0fa/atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TokenRangeResolution.java#L42-L43